downstream/modules/platform/con-controller-overview-details.adoc

```diff
@@ -0,0 +1,150 @@
+[id="con-controller-overview-details"]
+
+= Real-time playbook output and exploration
+{ControllerNameStart} enables you to watch playbooks run in real time, seeing each host as they check in. 
+You can go back and explore the results for specific tasks and hosts in great detail; search for specific plays or hosts and see just those results, or quickly zero in on errors that need to be corrected.
```

Comment:
Do you think this is fluff? "...or quickly zero in on errors that need to be corrected"

If so, I would re-write to something like "...or locate errors that need to be corrected."